test/flutter_test_infra/flutter_helpers.dart

```diff
@@ -2,47 +2,147 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// Helpers in this library are candidates to become separate
-// public package under https://github.com/flutter.
+// Content of this file is copied from
+// https://github.com/flutter/flutter/blob/master/packages/flutter/test/foundation/leak_tracking.dart
```

Comment:
instead of copying this file, can we just import this library from flutter? We could consider making this file part of flutter_test instead so that we can import it.

Reply:
We cannot because it is internal in Flutter, used in tests only for now, and we are not ready to publish it yet. 
In future it will be most likely published. Then we will remove it.

Added comment.

Reply:
Got it. Can you file an issue on leak_tracker and link in a comment here so that we do not lose track of this work?

Reply:
Added TODO. 

I do not think issue will make a difference, especially knowing that we will reorganize code, that will break all links in the issue: https://github.com/dart-lang/sdk/issues/52367

Issues for TODO make sense, when one is true:
1. There is a process that makes sure all issues are addressed timely (like in buganuzer)
2. There is discussion